docs/quick_start.md

```diff
@@ -48,4 +48,21 @@ Open broswer to visit
   - debugslo: [http://localhost:9097/d/lunettes-debugslo/lunettes-debugslo?orgId=1](http://localhost:9097/d/lunettes-debugslo/lunettes-debugslo?orgId=1)
 - jaejer: [http://localhost:9095/search](http://localhost:9095/search)
 - kibana: [http://localhost:9092](http://localhost:9092)
-- prometheus: [http://localhost:9091/graph?](http://localhost:9091/graph?)
\ No newline at end of file
+- prometheus: [http://localhost:9091/graph?](http://localhost:9091/graph?)
+
+## Update Lunettes with helm
+you can choose different lunettes version to upgrade
+```bash
+helm upgrade lunettes oci://registry-1.docker.io/lunettes/lunettes-chart --version [version] \
```

Comment:
we can use `helm upgrade --install` to upgrade or install

Reply:
done